monorepo/website/src/pages/news/2024-11-01-autumn-update.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Pathoplexus Autumn Update"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
+</div>
+
+# Pathoplexus Autumn Update
+
+*By the Pathoplexus Team - 1 November 2024*
+
+Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community (see 'past events' below). In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!
+
+As the seasons turn, we wanted to share some of our recent news:
+
+### In the media:
+- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)
+
+### Visitors and data:
+- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases
+- We are excited to have hit 100 registered users! (Necessary to upload data but not to view data)
```

Comment:
The parenthesis might be confusing. For example, one might think that 100 registered users are needed to upload data? What about this:

```suggestion
- We are excited to have hit 100 registered users! (Registration is needed to upload data but not to view data.)
```